bundle/src/main/java/com/cognifide/cq/cqsm/api/history/Entry.java

```diff
@@ -89,35 +90,31 @@
 	private final String path;
 
 	@Getter
-	private final String filePath;
+	private String filePath;
 
-	public Entry(Resource resource) {
-		path = resource.getPath();
-		filePath = getFilePath(resource);
-	}
+	@Getter
+	private Timestamp lastModified;

```

Comment:
Don't use Timestamp class. You should use classes rom new time api e.g. Instant.

Reply:
Replaced to java.util.Date type. It is enough for history row.